src/dags/stroomstoringen.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.postgres_operator import PostgresOperator
+from common import (
+    DATAPUNT_ENVIRONMENT,
+    SHARED_DIR,
+    MessageOperator,
+    default_args,
+    quote_string,
+    slack_webhook_token,
+)
+from common.db import DatabaseEngine
+from contact_point.callbacks import get_contact_point_on_failure_callback
+from http_fetch_operator import HttpFetchOperator
+from ogr2ogr_operator import Ogr2OgrOperator
+from postgres_permissions_operator import PostgresPermissionsOperator
+from postgres_rename_operator import PostgresTableRenameOperator
+from provenance_rename_operator import ProvenanceRenameOperator
+from sql.stroomstoringen import CONVERT_DATE_TIME
+
+dag_id: str = "stroomstoringen"
+variables: dict = Variable.get("stroomstoringen", deserialize_json=True)
+endpoint_url: str = variables["data_endpoints"]["stroomstoringen"]
+
+
+today = datetime.now().strftime("%m-%d-%Y")
+
+tmp_dir: str = f"{SHARED_DIR}/{dag_id}"
+total_checks: list = []
+count_checks: list = []
+geo_checks: list = []
+check_name: dict = {}
+db_conn: object = DatabaseEngine()
```

Comment:
This is correct, but also an extreme generalization (almost everything is an `object`).
This is better:
```python
db_conn: DatabaseEngine = DatabaseEngine()
```

Though:
```python
db_conn  = DatabaseEngine()
```
works equally well as MyPy can infer the type from the assignment.

Reply:
Ah didn't know that!